projects/addon-doc/interfaces/page.ts

```diff
@@ -16,7 +16,7 @@ export interface TuiDocPageGroup extends TuiDocPageBase {
     readonly subPages: readonly TuiDocPage[];
 }
 
-export type RawLoaderContent = Promise<{default: string}> | string;
+export type RawLoaderContent = Promise<{default: string}> | Promise<unknown> | string;
```

Comment:
Why do we need this?

Reply:
IDEA bugs

![image](https://github.com/taiga-family/taiga-ui/assets/12021443/d7ae4fd9-6e0f-4731-b38d-97db91ac5f75)